src/it/resources/org/checkstyle/suppressionxpathfilter/upperell/SuppressionXpathRegressionUpperEllSecond.java

```diff
@@ -0,0 +1,13 @@
+/*
+UpperEll
+
+*/
+
```

Comment:
can you please remove this comment? comments are ignored in this check anyway

Reply:
I was getting an error that "check name must be in the input file". For that reason, I have added the config

Reply:
I tried to remove it and the test passed. Maybe you got an error because filename was named incorrectly before and didn't have a check name in it. It shouldn't happen now because file name is correct now:
https://github.com/checkstyle/checkstyle/blob/1f519c204eaaa9eabe2b392f4b65abe00931991f/src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java#L299-L300
Can you please try to remove the comment and check if there is an error?

Reply:
Changes Done. You were right, I did a mistake in the class name. Thanks for the help!